src/relayFeeCalculator/chain-queries/zksync.ts

```diff
@@ -0,0 +1,28 @@
+import { DEFAULT_LOGGER, Logger } from "../relayFeeCalculator";
+import { providers } from "ethers";
+import { TOKEN_SYMBOLS_MAP } from "../../constants";
+import QueryBase from "./baseQuery";
+
+export class ZkSyncQueries extends QueryBase {
+  constructor(
+    provider: providers.Provider,
+    symbolMapping = TOKEN_SYMBOLS_MAP,
+    spokePoolAddress = "0xE0B015E54d54fc84a6cB9B666099c46adE9335FF",
+    usdcAddress = "0x0faF6df7054946141266420b43783387A78d82A9",
```

Comment:
Token address is wrong - should be 0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4.
```suggestion
    usdcAddress = "0x3355df6D4c9C3035724Fd0e3914dE96A5a83aaf4",
```